packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1626,7 +1626,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     if (!_didAutoFocus && widget.autofocus) {
       _didAutoFocus = true;
       SchedulerBinding.instance!.addPostFrameCallback((_) {
-        if (mounted) {
+        if (mounted && renderEditable.hasSize) {
```

Comment:
Should the text field not get focused when it has never been laid out, or should we just skip the `_updateSizeAndTransform` call but still allow the text field to get the focus?

Reply:
I prefer the former, it doesn’t make sense to get focus before layout(invisible). What do you think?